test/metric/stat.go

```diff
@@ -11,5 +11,5 @@ const (
 	SAMPLE_COUNT             Statistics = "SampleCount"
 	MINIMUM                  Statistics = "Minimum"
 	MAXUMUM                  Statistics = "Maxmimum"
-	HighResolutionStatPeriod            = 30
+	HighResolutionStatPeriod            = 10
```

Comment:
Reduce run time of metric tests

Reply:
Can you please check if this breaks any existing tests?
I think the statsd test might assume 30 seconds.
I know the `metric_aggregation_interval` in the JSON config is 30 seconds.
In general for any test that gets changed you should include screenshots or text output showing the test passes.

Reply:
I'd prefer to get input from other engineers about whether we should be reducing run time.
Is testing each feature for a minute or 2 really enough?

Was the reason to reduce test time due to GitHub limits or what?

Reply:
Added screen shots now.

Reply:
Why not just increase how long the credentials are valid?

The action:
https://github.com/aws-actions/configure-aws-credentials/tree/v2/
has a `role-duration-seconds` option.

Can we change the IAM role in AWS console as well?

Reply:
Or what if instead of a single GitHub "step" in a single GitHub "job" that runs a bunch of tests, we spread the tests across many GitHub jobs. If there are 4 jobs that each run 1/4 of the tests, and each job gets credentials at the start of the job, would that solve the problem?

Reply:
Good idea. I'll put in a pr to the gha. I still like reducing the test time. Furthermore, I dislike how long it takes to run tests. Not only that, but I feel like the shorter. The better.

Reply:
I still would like to keep this change if possible. Going from 30 minutes for the metric agent benchmark test per run to 20 is a win.

Reply:
Shorter test time makes developer cycle time for build/test/fix faster.
At some point we want longer running tests.
We do have a few "soak tests", but those don't cover every feature.

So if you feel that each of the features in the metric_value_benchmark suite are "sufficiently" tests by these 1 or 2 minute tests than I am fine keeping it that short.

Approving.